compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/SyntaxInfo.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.compiler.syntax;
+
+import io.ballerina.compiler.internal.parser.BallerinaLexer;
+import io.ballerina.compiler.syntax.tree.SyntaxKind;
+import io.ballerina.compiler.syntax.tree.Token;
+import io.ballerina.tools.text.CharReader;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * A class that provides syntax related information.
+ *
+ * @since 2.0.0
+ */
+public class SyntaxInfo {
+
+    /**
+     * Gives a list of all keywords in the ballerina.
+     *
+     * @return reserved keyword list
+     */
+    public static List<String> ballerinaKeywords() {
```

Comment:
Used `SyntaxInfo.ballerinaKeywords()` instead of `SyntaxInfo.getballerinaKeywords()` similar to `node.textRange()`, `node.children()`, `node.childEntries()` etc, 

If we want can add `get` prefix here. I'm okay with both.

Reply:
I think we can drop even `ballerina` part. `SyntaxInfo.keywords()` should be good enough IMO